Cargo.toml

```diff
@@ -8,5 +8,9 @@ edition = "2021"
 [dependencies]
 bevy = "0.9"
 opd-parser = { git = "https://github.com/ForesightMiningSoftwareCorporation/opd_parser" }
+anyhow = "*"
```

Comment:
In the event of a new major version, this can break the build. You should just choose `"1"`.